src/Methods/Address/GetSpendables.ts

```diff
@@ -9,29 +12,47 @@ import { btcToSat } from "../../Utilities/Bitcoin";
 
 const MAX_SPENDABLES = 200;
 
+async function outputHasRunes(outpoint: string): Promise<boolean> {
+  const [txid, n] = outpoint.split(":");
+  const tx = await getTransaction(txid);
+  if (!tx) return false;
+  const decipher = tryDecodeRunestone(tx);
+  if (!decipher) return false;
+  if (!isRunestone(decipher)) return false;
+
+  const pointer = decipher.pointer ?? 1;
+  if (decipher.etching || decipher.mint) {
+    if (Number(n) === pointer) return true;
+  }
+  if (decipher.edicts) {
+    decipher.edicts.forEach((edict) => {
+      if (Number(n) === edict.output) return true;
+    });
```

Comment:
i think this doesn't return true. `.forEach` doesnt have a return value and you can't break out of the loop early.

use something like `return decipher.edicts.some((edict) => edict.output === Number(n))` instead